adminSiteClient/VariablesAnnotationPage.tsx

```diff
@@ -147,7 +147,7 @@ const config: GrapherConfigGridEditorConfig = {
         version: 1,
         dimensions: [{ property: DimensionProperty.y, variableId: id }],
         map: {
-            variableId: id,
+            columnSlug: `${id}`,
         },
```

Comment:
Another option is to get rid of this here as the "default" for variable-configs entirely; the default for a chart is to use the first dimension for the map tab anyways.